ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -58,8 +65,6 @@ tests = testGroup "Mempool"
   , testProperty "addTxs txs == mapM (addTxs . pure) txs"  prop_Mempool_addTxs_one_vs_multiple
   , testProperty "result of addTxs"                        prop_Mempool_addTxs_result
   , testProperty "Invalid transactions are never added"    prop_Mempool_InvalidTxsNeverAdded
-  , testProperty "result of getCapacity"                   prop_Mempool_getCapacity
-  , testProperty "Mempool capacity implementation"         prop_Mempool_Capacity
```

Comment:
Draft because I temporarily removed these two tests. `prop_Mempool_Capacity` is a major headache to rewrite (I have a WIP lying around) because transactions now have dependencies. For example, removing a transaction can cause other transaction to be removed as well.

Idea: what if we add another transaction type (part of `GenTx`) that is totally independent (of the UTxO)? I bet that with such a transaction we can just restore the previous version of the test.

Reply:
Alternative approach: expose an internal variant of `addTxs` that instead of blocking, returns the added txs and the remaining txs. This function would be much easier to test. `addTxs` should then be implemented using this function. Downside: we're not testing the actual `addTxs`.

Reply:
I have taken the alternative approach: add a non-blocking variant of `addTxs`: `tryAddTxs` which is easily testable. This non-blocking variant is then used to implement `addTxs`.